languages/cpp/exercises/concept/numbers/.docs/after.md

```diff
@@ -0,0 +1,52 @@
+### Special integer types
+
+There are two other important integer types often used in the standard library:
+
+- `std::size_t`: unsigned integer type of the result of the `sizeof` operator as well as the `sizeof... operator` and the `alignof` operator (since C++11).
+- `std::ptrdiff_t`: is the signed integer type of the result of subtracting two pointers. The bit width of `std::ptrdiff_t` is not less than 17.
+
+### Integer literals
+
+Integer values can have different written form - "literal" that helps create integer numbers in different notations.
+
+- `10` - decimal number
+- `010` - octal number
+- `0xa` - hexadecimal number
+- `0b001` - binary number
+
+### Digit separators
+
+Long numbers in C++ can be separated for readability using `'` (single quotation mark). For example, `10000` is equivalent `10'000`.
+
+### Char types as integers
+
+Char types `char` `unsigned char`, `wchar_t`, `char16_t` and `char32_t` have the same memory layout and in some cases can be treated as integers.
```

Comment:
Indeed, your point is correct. I'll remove the section.

Answering 2, I have never seen `signed char` used anywhere, but `char` used for storing 'small' integers I have seen a lot.

Answering 3, this is also connected to using `char` as numeric type and doing arithmetic on it. It works because `char` is an integer type but it is not obvious in the semantic of character.